metrics/cli/docs/features/supported_storage_version/01_supported_storage_version.md

```diff
@@ -0,0 +1,80 @@
+
+### Requirements
+> Define requirements and make sure that they are complete.
+
+The Metrics CLI supported storage version feature has the following requirements:
+
+- The Metrics CLI should pass the supported storage version to the `Metrics Web` application during the building process.
+- The Metrics CLI should set the supported storage version of the `Metrics Web` application into the application's environment variable named `SUPPORTED_STORAGE_VERSION`.
+- The Metrics CLI has to get the supported storage version from the `STORAGE_VERSION` file under the `metrics` project folder.
+
+### Landscape
+> Look for existing solutions in the area.
+
+According to the [Supported Storage Version](https://github.com/Flank/flank-dashboard/blob/master/metrics/docs/01_storing_storage_metadata.md#supported-storage-version) section of the `Storing Storage Metadata` document, we are storing the storage version in the `STORAGE_VERSION` file under the `metrics` package. Also, on the stage of the application deployment, the storage version and the supported storage version are the same. More precisely, the `STORAGE_VERSION` holds the current storage version that is compatible with all applications in the `metrics` folder at the current point of the repository history.
+
+Once we've figured out the place of storing the storage version, let's take a look at how to pass this version to the `Metrics Web` application. As the [Storing Storage Metadata](https://github.com/Flank/flank-dashboard/blob/master/metrics/docs/01_storing_storage_metadata.md#supported-storage-version) section states, the `Metrics Web` application holds its supported storage version as an environment variable, so we should pass it to the `Metrics Web` application during the building process using the `dart-define` flag.
+
+Since the storing of the supported storage versions is pretty custom, we are going to use the custom solution for setting this version in the `Metrics CLI`. 
+
+### Prototyping
+> Create a simple prototype to confirm that implementing this feature is possible.
+
+So, to provide a supported storage version to the `Metrics Web` application during the deployment process, we should follow the next steps:
+
+1. Fetch the storage version from the `STORAGE_VERSION` file under the `metrics` package.
+2. Pass the fetched storage version to the `Metrics Web` application environment as a `supported storage version` during the application building process.
+
+Let's consider the code snippets proving that this feature is possible to implement:
+
+To read the data from the file, we are going to use the [File](https://api.dart.dev/stable/2.13.1/dart-io/File-class.html) class, providing an ability to get the file content using the following code:
```

Comment:
:label:
We can actually avoid setting a version on URL so the reader would always navigate to the latest version.
```suggestion
To read the data from the file, we are going to use the [File](https://api.dart.dev/stable/dart-io/File-class.html) class, providing an ability to get the file content using the following code:
```